content/blog/2017-11-11-rperseus.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Exploratory Data Analysis of Ancient Texts with rperseus"
+categories: blog
+date: '2017-11-17'
```

Comment:
Post date will be 2017-11-28. Please update here and in image filenames

Reply:
Updated.